interfaces/MatlabToolbox/matlab_new/Phases/GRI30.m

```diff
@@ -0,0 +1,47 @@
+function gas = GRI30(tr)
```

Comment:
The dedicated object representing GRI3.0 have been deprecated and removed from the Python API